sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -53,47 +60,253 @@ def _is_ipython(self):
         """
 
-        if namespace is None:
-            namespace = utils.set_katib_namespace(exp_object)
         try:
-            outputs = self.api_instance.create_namespaced_custom_object(
+            self.api_instance.create_namespaced_custom_object(
                 constants.KUBEFLOW_GROUP,
                 constants.KATIB_VERSION,
                 namespace,
                 constants.EXPERIMENT_PLURAL,
-                exp_object)
+                exp_object,
+            )
         except client.rest.ApiException as e:
             raise RuntimeError(
                 "Exception when calling CustomObjectsApi->create_namespaced_custom_object:\
-         %s\n" % e)
+         %s\n"
+                % e
+            )
+
+        # TODO (andreyvelich): Use proper logger.
+        print("Experiment {} has been created".format(exp_object.metadata.name))
 
         if self._is_ipython():
             if self.in_cluster:
                 import IPython
-                IPython.display.display(IPython.display.HTML(
-                    'Katib Experiment {} '
-                    'link <a href="/_/katib/#/katib/hp_monitor/{}/{}" target="_blank">here</a>'.format(
-                        exp_object.metadata.name, namespace, exp_object.metadata.name)
-                ))
-        return outputs
+
+                IPython.display.display(
+                    IPython.display.HTML(
+                        "Katib Experiment {} "
+                        'link <a href="/_/katib/#/katib/hp_monitor/{}/{}" target="_blank">here</a>'.format(
+                            exp_object.metadata.name,
+                            namespace,
+                            exp_object.metadata.name,
+                        )
+                    )
+                )
+
+    def tune(
+        self,
+        # TODO (andreyvelich): How to be consistent with other APIs (name) ?
+        name: str,
+        objective: Callable,
+        parameters: Dict[str, Any],
+        base_image: str = constants.BASE_IMAGE_TENSORFLOW,
+        namespace: str = utils.get_default_target_namespace(),
+        algorithm_name: str = "random",
+        objective_metric_name: str = None,
+        additional_metric_names: List[str] = [],
+        objective_type: str = "maximize",
+        objective_goal: float = None,
+        max_trial_count: int = None,
+        parallel_trial_count: int = None,
+        max_failed_trial_count: int = None,
+        retain_trials: bool = False,
+        packages_to_install: List[str] = None,
+        pip_index_url: str = "https://pypi.org/simple",
+    ):
+        """Create HyperParameter Tuning Katib Experiment from the objective function.
+
+        Args:
+            name: Name for the Experiment.
+            objective: Objective function that Katib uses to train the model.
+                This function must be Callable and it must have only one dict argument.
+                Katib uses this argument to send HyperParameters to the function.
```

Comment:
I think we need to strongly-specify that objective function should inner-import statements for all necesary packages

Reply:
Let's discuss about it in this comment: https://github.com/kubeflow/katib/pull/1951#discussion_r972199141.
At the minimum, we can just extend this comment as you mentioned.